.vscode/settings.json

```diff
@@ -3,7 +3,7 @@
     "editor.defaultFormatter": "ms-python.black-formatter",
     "editor.formatOnSave": true,
     "editor.codeActionsOnSave": {
-      "source.organizeImports": true
+      "source.organizeImports": "explicit"
```

Comment:
what does this do?

Reply:
The configuration for organizing imports in our code editor changed automatically from true to "explicit". This likely occurred due to an update or default setting adjustment within the development environment.